app/players/dwarves/dwarf_blacksmith.py

```diff
@@ -0,0 +1,18 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfBlacksmith(Dwarf):
+
+    def __init__(self, nickname: str,
```

Comment:
Don't forget to start every argument from a new line(`nickname` should be on a new line. Refactor everywhere